### PR TITLE
Add ha+redis-operator test scenario for SPR:2.1

### PR DESCRIPTION
**Pull request by flaviodsr**
This PR also includes other changes required for the HA+redis-operator test scenario.

### CODE REVIEW

File: .github/workflows/cluster-rotate.yml [hunk 1]
```diff
@@ -161,8 +166,20 @@ jobs:
...
+                done
+                flags=""
+                ;;
+              redisoperator)
+                flags=""
+                export HELM_EXPERIMENTAL_OCI=1
+                helm chart pull "${REDIS_OPERATOR_CHART_SRC}"
+                helm chart export "${REDIS_OPERATOR_CHART_SRC}"
+                # replace images repository according to source
+                sed -i "s,registry.suse.com,${REDIS_OPERATOR_CHART_SRC/charts*}containers,g" ${service}/values.yaml
```
Comment (jsuchome) [inline .github/workflows/cluster-rotate.yml:182]: Why do you have to use internal images?

Comment (flaviodsr) [reply .github/workflows/cluster-rotate.yml:182]: Because `redis-operator` is installed from the Devel repository which is not published outside.

Comment (jsuchome) [reply .github/workflows/cluster-rotate.yml:182]: Why can't we use the public one? We didn't publish it yet?

Comment (flaviodsr) [reply .github/workflows/cluster-rotate.yml:182]: Its just that we want to catch issues as early as possible and Devel is the earlier we can go. If we choose to use the published images we risk finding issues only after releasing.

Comment (jsuchome) [reply .github/workflows/cluster-rotate.yml:182]: OK, good point

**Review by jsuchome — APPROVED** [review]

**Pull request merged**